tests/test_routing.py

```diff
@@ -43,3 +45,23 @@ def test_request_with_trailing_slash_not_redirected(
     """
     response = test_app.get(invalid_route)
     assert response.status_code == 404
+
+
+@pytest.mark.parametrize(
+    "test_root_path,expected_status_code",
+    [("", 200), ("/api/v1", 200), ("/wrongroot", 404)],
+)
+def test_docs_work_using_defined_root_path(
+    test_app, test_root_path, expected_status_code, monkeypatch
+):
```

Comment:
```suggestion
def test_docs_work_using_defined_root_path(
    test_app, test_root_path, expected_status_code, monkeypatch
):
    """
    Ensure that the interactive docs are reachable both with and
    without the root_path prefix predefined in the app
    """
```
I find the purpose of this test hard to follow. Clearer variable names could help, but a docstring is also needed.